src/main/resources/db/migration/V1__BasicDBSetup.sql

```diff
@@ -0,0 +1,51 @@
+create table if not exists category
```

Comment:
table names should be in plural

Reply:
Yep, I know, but for simplicity and due to the fact, that some plurals in english cannot be created the regular way, by just adding -s to the word (e.g. person -> people), creators of Hibernate decided to use singular form... So, to avoid use of accotations above the classes with table names, we will just use the regular for Hibernate format - singular form.

Reply:
actually, there is there is no final opinion regarding this dilemma